src/main/java/at/hannibal2/skyhanni/features/inventory/MenuItemDisplayOverlayAbiphone.kt

```diff
@@ -60,7 +60,7 @@ class MenuItemDisplayOverlayAbiphone {
         if ((stackSizeConfig.contains(1)) && (chestName.contains("Abiphone"))) {
             if (item.name == null) return ""
             val lore = item.getLore()
-            if ((item.name?.startsWith("§f§")) && (lore.any { it.lowercase().contains("§cdo not disturb") }) && lore.any { it.lowercase().contains("enabled!") }) {
+            if ((item.name.startsWith("§f§")) && (lore.any { it.lowercase().contains("§cdo not disturb") }) && lore.any { it.lowercase().contains("enabled!") }) {
```

Comment:
Please don't use lowercase in that situation. Just change the text you compare to correct upper case letters.

Reply:
ill do it to just this scenario; knowing hypixel admins theyre going to break the "SkyBlock Menu" capitalization for april fools day